scripts/ingest/insert_elasticsearch.py

```diff
@@ -31,32 +33,15 @@ def random_ip():
 # Fill in with events you want to write to elasticsearch
 # NEED TO MODIFY
 events = [
```

Comment:
Can you re-add the example formatted event from before, so we know what format this event should be in when using it? The existing example event is also an event that will trigger an alert in mozdef docker environment, so having it hardcoded in the file in SCM is also useful.